frontend/packages/dev-console/src/components/helm/HelmReleaseDetailsPage.tsx

```diff
@@ -52,7 +54,16 @@ export const LoadedHelmReleaseDetailsPage: React.FC<LoadedHelmReleaseDetailsPage
   if (_.isEmpty(secretResource)) return <ErrorPage404 />;
 
   const secretName = secretResource[0]?.metadata.name;
-  const releaseName = secretResource[0]?.metadata.labels?.name;
+  const releaseName = helmReleaseData?.name;
+
+  const title = (
+    <>
+      {releaseName} &nbsp;
```

Comment:
should we not use CSS instead?

Reply:
If it's just a simple space that we need then I don't mind using this HTML entity.